### ISSUE TITLE
CustomMouse not positioning correctly with TextBox2

### ISSUE THREAD

**Issue by TEParsons** [psychopy/psychopy#3425]
Came across this while editing the customMouse.py demo to use Textboxes, can be seen in this minimal working example:

```
from psychopy import visual, event

# Make window
win = visual.Window(units="height")
# Make custom mouse
vm = visual.CustomMouse(win,
    leftLimit=-0.2, topLimit=0.2, rightLimit=0.2, bottomLimit=-0.2,
    showLimitBox=True, clickOnUp=True)
# Make pointer
new_pointer = visual.TextBox2(win, 
    text=u'\u265e', 
    font="Gothic A1")
# Set pointer
vm.pointer = new_pointer

while not event.getKeys():
    # Draw components
    vm.draw()
    win.flip()
```

**Comment by isolver** [comment]
Looks like issue is two fold:
1) CustomMouse only works with 'norm' units - if window is set to any other unit type placement of any stim seems wrong.
2) Textbox2 does not support norm units, only 'height'?.
May need to leave this one until next release.

**Comment by TEParsons** [comment]
Fixed in 2022.2.x, as Textbox2 can now have `norm` units and can set its anchor and alignment. Only thing to keep in mind is that the textbox should be anchored and aligned to the centre when doing this!
